server/db/migration/survey/migrations/sqls/20181204101541-create-history-log-table-up.sql

```diff
@@ -1,11 +1,11 @@
 CREATE TABLE activity_log
 (
-  id           BIGSERIAL NOT NULL,
+  id           bigint       NOT NULL GENERATED ALWAYS AS IDENTITY,
   type         VARCHAR(255) NOT NULL,
-  user_uuid    uuid NOT NULL,
-  content      jsonb NOT NULL DEFAULT '{}'::jsonb,
-  system       boolean NOT NULL DEFAULT false,
-  date_created TIMESTAMP WITHOUT TIME ZONE NOT NULL DEFAULT (now() AT TIME ZONE 'UTC'),
```

Comment:
Why have you removed "WITHOUT TIME ZONE"?

Reply:
TIMESTAMP is equivalent to WITHOUT TIME ZONE. `TIMESTAMPTZ` corresponds to `WITH TIME ZONE`

Reply:
right, this is what PostgreSQL documentation says: The SQL standard requires that writing just timestamp be equivalent to timestamp without time zone, and PostgreSQL honors that behavior. timestamptz is accepted as an abbreviation for timestamp with time zone; this is a PostgreSQL extension.